core/src/test/java/org/mobilitydata/gtfsvalidator/parsing/CsvRowTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.parsing;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.Reader;
+import java.io.StringReader;
+
+import static com.google.common.truth.Truth.assertThat;
+
+@RunWith(JUnit4.class)
+public class CsvRowTest {
+
+    @Test
+    public void shouldReturnRowNumber() {
+        Reader reader = new StringReader("stop_id,stop_name,stop_lat" + System.lineSeparator() +
```

Comment:
No need to use System.lineSeparator() - CSV parser must support all kinds of line delimiters. Please use the normal "\n".